doc/api/fs.md

```diff
@@ -910,7 +910,7 @@ changes:
 * `options` {Object|string}
   * `encoding` {string|null} **Default:** `'utf8'`
   * `mode` {integer} **Default:** `0o666`
-  * `flag` {string} **Default:** `'a'`
+  * `flag` {string} See [support of file system `flags`][] **Default:** `'a'`
```

Comment:
It seems we have two full sentences here now, so we need to add periods after the both:

`` See [support of file system `flags`][]. **Default:** `'a'`. ``

The same for all the cases below.

Reply:
Resolved in commit 14da387a73a827670e5c379da727bbb489b93e3b